src/module/meme-object.js

```diff
@@ -0,0 +1,65 @@
+let memeList = [];
+// === set local storage === //
+const setLocal = () => localStorage.setItem('memelist', JSON.stringify(memeList));
+
+// === get local storage === //
+const getLocal = () => {
+  const data = JSON.parse(localStorage.getItem('memelist'));
+  if (data !== null) {
+    return data;
+  }
+  return [];
+};
+
+// === update the index values when delete an item === //
+const updateindex = () => {
+  memeList.forEach((element, index) => {
+    element.index = parseInt(index, 10) + 1;
+    setLocal();
+  });
+};
+
+// === update the meme text value === //
+const updateText = (id, text) => {
+  memeList = getLocal();
+  id -= 1;
+  memeList[id].description = text;
+  setLocal();
+};
+
+// === delete item from meme === //
+const del = (index) => {
+  index = Number(index);
+  memeList = getLocal();
+  memeList = memeList.filter((e) => e.index !== index);
```

Comment:
Please consider changing the "e" for the complete word event.

Reply:
thank you 😃